client/src/components/BookList.tsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react"
+import BookCard from "./BookCard"
+
+const BookList = () => {
+
+    let [booksData, setBooksData] = useState([])
+
+    const getBookData = () => {//http://localhost:8080/books/list
+        console.log(process.env.REACT_APP_SERVER_IP)
```

Comment:
good job getting this working

Typical convention here would be to assign this to a URL at the top of the file, so

on line 3 or 4

`const url = process.env.REACT_APP_SERVER_IP`

then use string templates as shown

Reply:
Changed that and used template strings (and got rid of the comments).